tools/ci/launch_cluster.sh

```diff
@@ -17,7 +17,7 @@ while [ x"${LAUNCH_SUCCESS}" == x"False" ]; do
     else
         set -e
     fi
-    /venvs/wrap.sh dcos-launch wait --info-path=${CLUSTER_INFO_FILE} 2>&1 | tee dcos-launch-wait-output.stdout
+    /venvs/wrap.sh dcos-launch dcos-launch wait --info-path=${CLUSTER_INFO_FILE} 2>&1 | tee dcos-launch-wait-output.stdout
```

Comment:
Please add a comment here, explaining that the first parameter is the environment name and *NOT* the command :)

Reply:
https://github.com/mesosphere/dcos-commons/pull/3111/commits/63d62c9116066e5a3b18512fd2b211befd32a9a8